src/amo/components/LandingPage/index.js

```diff
@@ -37,17 +37,23 @@ const ICON_MAP = {
 
 export class LandingPageBase extends React.Component {
   static propTypes = {
-    addonTypeOfResults: PropTypes.string,
+    // This is a bug; addonTypeOfResults is used in
+    // `componentWillReceiveProps()`.
+    // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
I say disable it at the top -- it's annoying and full of bugs! If you search for components that disable it, you'll find a lot.

Reply:
But then we lose out on it for the entire file and it's still useful more than it isn't. Removing it means we'll miss out on opportunities to remove unused prop-types in the future, which is handy.

Reply:
porting the prop types to Flow will get us much better coverage of prop usage.